api/client/src/main/java/org/projectnessie/client/auth/oauth2/AuthorizationCodeFlow.java

```diff
@@ -103,7 +103,7 @@ class AuthorizationCodeFlow extends AbstractFlow {
         redirectUriFuture
             .thenApply(this::extractAuthorizationCode)
             .thenApply(this::fetchNewTokens)
-            .whenComplete(this::log);
+            .whenComplete((tokens, error) -> log(error));
```

Comment:
What's the warning here? Old code LGTM :thinking:

Reply:
`tokens` parameter removed - so IJ refactored it to a lambda